math/vampire/Portfile

```diff
@@ -1,24 +1,23 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           github 1.0
 PortGroup           cmake 1.1
+PortGroup           github 1.0
 
 # Until a new release is published under the 3-clause BSD license, we have to
 # track the development branch; as of this writing, the last stable
 # release (4.5.1) was made under the previous, restricted-use license.
```

Comment:
This comment no longer applies; you could remove this or rewrite it to explain the new reason why we're not using a release.

Reply:
Done